casbin/enforcer.cpp

```diff
@@ -78,7 +79,7 @@ bool Enforcer :: enforce(string matcher, Scope scope) {
     int policy_len = int(this->model->m["p"].assertion_map["p"]->policy.size());
 
     vector <Effect> policy_effects(policy_len, Effect :: Indeterminate);
-    vector <float> matcher_results(policy_len);
+    vector <float> matcher_results(policy_len, 0.0);
```

Comment:
0.0 or 1.0?

Reply:
why 1.0 ?

Reply:
Init with 0.0 (not matched) seems OK to me.